packages/xo-web/src/common/xo/index.js

```diff
@@ -424,7 +424,10 @@ export const subscribeNotifications = createSubscription(async () => {
 
   let notifications
   try {
-    notifications = await updater._call('getMessages')
+    const now = Date.now()
```

Comment:
`Date.now` can be inlined.

Reply:
It would be called "many" times with slightly different dates for each comparison, though.